docs/using/tsuru.yaml.rst

```diff
@@ -10,9 +10,9 @@ tsuru.yaml
 tsuru.yaml is a special file located in the root of the application. The name of
 the file may be ``tsuru.yaml`` or ``tsuru.yml``.
 
-This file is used to describe certain aspects of your app. Currently it describes
+This file is used to describe certain aspects of your app. Currently, it describes
 information about deployment hooks and deployment time health checks. How to use
-this features is described below.
+these features is described below.
```

Comment:
there are still grammar errors in here... it should be "these features ARE described bellow"